includes/class-coblocks-form.php

```diff
@@ -974,7 +973,6 @@ public function process_form_submission( $atts ) {
 		$email_headers = (array) apply_filters(
 			'coblocks_form_email_headers',
 			array(
-				"From: {$from_name}",
 				"Reply-To: {$reply_to}",
```

Comment:
@EvanHerman I think we shouldn't scrap the senders name along with the `From` header. It seems like we should do `"Reply-To: {$sender_name} <{$sender_email}>"`